td.server/src/config/parsers.config.js

```diff
@@ -1,18 +1,12 @@
-'use strict';
+import cookieParser from 'cookie-parser';
+import express from 'express';
 
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-
-var parsers = function (app) {
-   
-    app.use(bodyParser.json());
-    app.use(bodyParser.urlencoded({ extended: false }));
-    app.use(cookieParser());  
-    
+const config = (app) => {
```

Comment:
body-parser has been depreciated.  The official recommendation is to use `express.json()` and `express.urlencoded` functions.  I removed the body-parser dependency as well.